lib/rubocop/config.rb

```diff
@@ -80,7 +80,7 @@ def cop_enabled?(cop)
     def warn_unless_valid
       validate
     rescue Config::ValidationError => e
-      warn "Warning: #{e.message}".color(:red)
+      warn Rainbow.new.wrap("Warning: #{e.message}").red
```

Comment:
Need to figure out why I keep getting `undefined method `Rainbow' for module `Kernel'` here when I try to call `Rainbow("Warning: #{e.message}")`,